corona_stats/README.md

```diff
@@ -1,4 +1,4 @@
-# Coronaviruses
+# Get live Coronavirus stats in Slack with Nimbella Commander
```

Comment:
```suggestion
# Coronavirus Stats

Get live COVID-19 stats in slack with Nimbella Commander. 
```
What do you think about this?

Reply:
some crawlers use the first line as description, we can let it be a bit long as it gives complete sense.

Reply:
Okay